scripts/gen-github-release.sh

```diff
@@ -0,0 +1,165 @@
+  COSMOS_SDK_VERSION="$(version-from-go-mod cosmos-sdk '$COSMOS_SDK_VERSION')" \
+  COMETBFT_VERSION="$(version-from-go-mod cometbft '$COMETBFT_VERSION')"
+cat << 'EOF' | awk "$AWK_REPLACE_PLACEHOLDERS" "$@"
+The Agoric OpCo engineering team is pleased to publish the **`$TAG`** release. This release is primarily intended to $REASON.
+
+The following new features are included in this release:
+
+- #$ISSUE: $DESCRIPTION
+
+The release contains at least the following fixes:
+
+- #$ISSUE: $DESCRIPTION
+
+The full set of changes in this release can be found at https://github.com/Agoric/agoric-sdk/pull/$RELEASE_PR.
+
+This code has satisfied all pre-release/testnet validation checks, and is now recommended for nodes to upgrade from the previous `$PREV_RELEASE` release. As a state-machine-breaking upgrade, once approved, all chain validators will need to upgrade from `$PREV_RELEASE` to this new version (after the chain halts due to reaching the height required in a governance proposal).
+
+Since the `agoric-upgrade-11` release, state-sync snapshots include more data than before. Nodes which have inadvertently pruned this data (e.g. those created from a state-sync before the `agoric-upgrade-11` release) will not be able to produce such snapshots, and will need to be restored from state-sync. We are aware of continued performance issues related to state-sync. In particular, we've observed that on some deployments, the current implementation can require 100 GB of temporary free disk space and 16GB of memory.
+
+Below is the _cosmos upgrade handler name_ for this release. This is the name that can be used in governance proposals to deploy this upgrade.
+
+```
+Cosmos Upgrade Handler Name: $TAG
+```
+
+Below is the git information related to this software release. Note the _git tag_ and _cosmos upgrade handler name_ differ.
+
+```
+Git Tag: $TAG
+Git Commit: $COMMIT
+@agoric/cosmos package version: $AGORIC_COSMOS_VERSION
+Docker: ghcr.io/agoric/agoric-sdk:$DOCKER_IMAGE_TAG
+```
+
+As shown in go.mod this release is based on:
+
+```
+ibc-go $IBC_GO_VERSION
+cosmos-sdk $COSMOS_SDK_VERSION
+cometbft $COMETBFT_VERSION
+```
+
+## How to upgrade
+
+Presuming that your node is running `$PREV_RELEASE`, once the upgrade height for a subsequent proposal to upgrade to `$TAG` has been reached, your node will halt automatically allowing you to upgrade the agoric stack.
+
+```
+# (prepare by installing Go 1.20.2 or higher, Node 16 or 18, clang 10 or gcc 10)
+# (stop the agd service)
+cd agoric-sdk
+git fetch --all
+git checkout $TAG
+git clean -xdf
+yarn install
+yarn build
+(cd packages/cosmic-swingset && make)
+# (start the agd service)
+```
+
+Do _not_ copy the `agd` script or Go binary to another location. If you would like to have an executable `agd` in another location, then create a symlink in that location pointing to `agoric-sdk/bin/agd`.
+
+### Golang Version
+
+The `$TAG` release requires Go 1.20.2 or higher.
```

Comment:
This Go minimal version number is getting replicated in too many places. Consider referring to, e.g. README.md at the tag version. Same comment for node and C compiler versions.